scene/3d/interpolated_camera.cpp

```diff
@@ -41,7 +41,7 @@ void InterpolatedCamera::_notification(int p_what) {
 				set_physics_process(false);
```

Comment:
Do you know what this is used for? Seems weird in a class that uses (internal) process to disabled physics process.

Reply:
`git blame` says it's been there forever, with no reason that I can think of. I'd say just kill it as part of this commit too.

Reply:
According to line 112-113 I think it's to disable it if it's running in the editor, so I guess this should be `set_process_internal(false)`